### PR TITLE
Issue #3. Move id string generation to its own module for re-usability.

### PR DESCRIPTION
**Pull request by mtuchowski**

### CODE REVIEW

**Comment by mtuchowski** [comment]
Ups.... fix pending...

File: html-document/to-id-string.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var startsWith = require('es5-ext/string/#/starts-with')
```
Comment (medikoo) [inline html-document/to-id-string.js]: Let's add some tests to it

**Comment by medikoo** [comment]
For future PR's in those projecst can also by assigned to issues via hub ;-).

So let's have tests, and we'll take it

**Comment by medikoo** [comment]
It's also merged to eregistrations-salvador

**Pull request merged**